test/unit/com/datastax/driver/core/PreparedStatementHelper.java

```diff
@@ -41,16 +41,6 @@ private static MD5Digest id(PreparedStatement statement)
         return statement.getPreparedId().boundValuesMetadata.id;
     }
 
-    public static void assertStable(PreparedStatement first, PreparedStatement subsequent)
```

Comment:
Moved this to test code (since only tests rely on this class), and also removed this method which is unused.